src/api/app/controllers/webui/package_controller.rb

```diff
@@ -81,6 +81,7 @@ def show
 
     @comments = @package.comments
     @requests = []
+    @services = Service.find(project: @project.name, package: @package.name)
```

Comment:
BTW, does it worth to implement something like this? I mean, to have a .exists? method at ActiveXML::Transport or ActiveXML::Node. Looks like is better answering with a boolean instead of retrieving the whole object from the backend (despite the rails cache).